src/components/common/AnimationControls.js

```diff
@@ -26,10 +28,18 @@ import {
 } from '../../config/constants';
 
 const useStyles = makeStyles(() => ({
+  buttonContainer: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  button: {
+    fontSize: '1.75em',
```

Comment:
I prefer `rem`, but up to you.

For more on `em` vs `rem`, see:

https://webdesign.tutsplus.com/tutorials/comprehensive-guide-when-to-use-em-vs-rem--cms-23984

Reply:
Very interesting article. Did not know all those nuances. I'm going to leave as `em` for now to be consistent with the other apps/not have to determine the `rem` equivalent of `1.75em`.